File: application/src/main/java/com/example/server/dto/BookingDto.java
package com.example.server.dto;

import com.example.api.dto.ItemDto;
import com.example.api.dto.UserDto;
import com.example.server.repository.entity.BookingStatus;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class BookingDto {
    private final Long id;
    private final LocalDateTime start;
    private final LocalDateTime end;
    private final ItemDto item;
    private final UserDto booker;
    private final BookingStatus status;
}

File: api/src/main/kotlin/com/example/api/model/BookingDto.kt
package com.example.api.model

import io.swagger.v3.oas.annotations.media.Schema
import java.time.LocalDateTime

@Schema(description = "Информация о запросе на бронирование")
data class BookingDto(

    @get:Schema(description = "Идентификатор запроса на бронирование")
    val id: Long,

    @get:Schema(description = "Дата и время начала бронирования")
    val start: LocalDateTime,

    @get:Schema(description = "Дата и время окончания бронирования")
    val end: LocalDateTime,

    @get:Schema(description = "Информация о предмете")
    val item: ItemDto,

    @get:Schema(description = "Информация о пользователе")
    val booker: UserDto,

    @get:Schema(description = "Статус бронирования")
    val status: BookingStatus
)
